contracts/base/Dispatcher.sol

```diff
@@ -181,8 +181,12 @@ abstract contract Dispatcher is Payments, V2SwapRouter, V3SwapRouter, Callbacks
                         (address token, address recipient, uint256 id, uint256 amount) =
                             abi.decode(inputs, (address, address, uint256, uint256));
                         Payments.sweepERC1155(token, recipient.map(), id, amount);
+                    } else if (command == Commands.SEAPORT_V2) {
```

Comment:
any reason to keep support for the previous Seaport in?

Reply:
I think while they plan on using the same conduit for Seaport1.2 as the previous one, 0age mentioned they will be removing the old version 1.1 in the future:
```
If you're currently building on v1.1, you're welcome to keep using it (though it will eventually be removed from 
[@OpenSea](https://twitter.com/opensea)
's conduit)
```
https://twitter.com/z0age/status/1621240001114836992